application/controllers/handler.js

```diff
@@ -839,13 +841,17 @@ module.exports = {
           }
 
           let dCreator = document.creator.userid || document.creator;
-          if (dCreator !== group.creator.userid) {
+          let idAdmin = !document.members.reduce((a, c) => {
```

Comment:
Code is really not clear here, you use double negative and really lost me. You should comment stuff like this. I think there's a typo and it should be `isAdmin`? In that case, I would expect something like `document.members.some((c) => c.userid === userid && c.role === 'admin')`??? But I'm not sure that's what you want.

Reply:
Yes a typo and too complicated code. I wasnt aware of the existance of Array.some(). Thanks for suggesting!